design.md

```diff
@@ -0,0 +1,4 @@
+- Add CommitQuorum to CreateIndexesOptions 
```

Comment:
I'm guessing you didn't mean to add this file to the PR.

Reply:
Whoops. Removed.